demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -751,6 +753,17 @@ export class DemoMeetingApp
       );
     });
 
+    if(!this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback() || !this.enableUnifiedPlanForChromiumBasedBrowsers)  {
```

Comment:
Can this check move to a private method?

Reply:
Done